.teamcity/src/subprojects/build/core/NativeBuild.kt

```diff
@@ -18,6 +19,10 @@ class NativeBuild(private val osEntry: OSEntry) : BuildType({
             jdkHome = "%env.JDK_11%"
         }
     }
+    features {
```

Comment:
Code duplication

Reply:
Yes. Each block has this and it's required and if it diverges will add contents there